docker-compose.armhf.yml

```diff
@@ -35,7 +35,7 @@ services:
 
     # Docker Swarm provider
     faas-swarm:
-        image:  openfaas/faas-swarm:0.4.4-armhf
+        image:  openfaas/faas-swarm:0.5.0-armhf
```

Comment:
Hmm I tried `docker pull openfaas/faas-swarm:0.5.0-armhf` and it didn't work. I'll look into this before merging.

Reply:
Per my comment in the PR, I can't build and push this image, I assume that you will build and push it